themes/default/content/blog/pulumi-targets-dotnet-6/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Pulumi now targets .NET 6"
+authors: ["zaid-ajaj"]
+tags: ["dotnet", "csharp", "fsharp", "vb.net"]
+meta_desc: "Introducing a static-code analyzer for C# which provides instant feedback on common mistakes defining Pulumi resources"
+# meta_image: enhanced-static-code-analysis.png
+date: "2022-07-20"
+---
+
+In this blog post, we will talk about how Pulumi is now using [.NET 6](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-6) as our default across the ecosystem. We will discuss the changes applied to templates, program structure and code generation. We also explain how Pulumi C# projects can benefit from latest features in .NET 6 and how it simplifies your programs overall. Let's dive in, shall we?
```

Comment:
I would try to summarize our motivation for the change - .NET 6 is now the latest LTS version and it allows us to use a cleaner version of C#